dev/src/__tests__/updateEnrollment.test.tsx

```diff
@@ -0,0 +1,92 @@
+import { Enrollment, User } from "../../proto/ag/ag_pb"
+import { createOvermindMock } from "overmind"
+import { config } from "../overmind"
+import { configure, render } from "enzyme"
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17"
+import { createMemoryHistory } from "history"
+import React from "react"
+import Members from "../components/Members"
+import { Route, Router } from "react-router"
+import { Provider } from "overmind-react"
+import { MockGrpcManager } from "../MockGRPCManager"
+
+
+React.useLayoutEffect = React.useEffect
+
+describe("UpdateEnrollment", () => {
+    const mockedOvermind = createOvermindMock(config, {
+        grpcMan: new MockGrpcManager()
+    })
+    it("Pending student gets accecpted", async () => {
+        await mockedOvermind.actions.getEnrollmentsByCourse({ courseID: 2, statuses: [] })
+        //This is a user with status pending
+        window.confirm = jest.fn(() => true)
+        mockedOvermind.actions.updateEnrollment({ enrollment: mockedOvermind.state.courseEnrollments[2][1], status: Enrollment.UserStatus.STUDENT })
```

Comment:
The code `mockedOvermind.state.courseEnrollments[2][1]` seems to be an enrollment. It is also used on Line 25. Could it be made into a separate variable, e.g.,:
```typescript
enrollment = mockedOvermind.state.courseEnrollments[2][1]
```
That can be used in both Lines 24 and 25.

Reply:
Similar for the remaining tests below.